spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java

```diff
@@ -1635,13 +1598,10 @@ public void foundGuaranteedNullDeref(@Nonnull Set<Location> assignedNullLocation
 
             if (sourceLocations == doomedLocations && assignedNullLocationSet.size() == 1) {
                 Location assignedNull = assignedNullLocationSet.iterator().next();
-                SourceLineAnnotation sourceLineAnnotation = SourceLineAnnotation.fromVisitedInstruction(classContext, method,
-                        assignedNull);
-                if (sourceLineAnnotation != null) {
```

Comment:
Removing the null check, since `SourceLineAnnotation.fromVisitedInstruction()` won't return null.